README.md

```diff
@@ -1,2 +1,11 @@
 # software_extras
 Documentation, scripts and howtos for BGH software
+
+`raijin_install` should be used to install the latest stable version of Underworld / UWGeodynamics / Badlands
```

Comment:
To make it explicit add "for versions of UWGeodynamics up to and including v2.8"